ui/src/saros/stf/server/bot/jquery/JQueryHelper.java

```diff
@@ -0,0 +1,100 @@
+package saros.stf.server.bot.jquery;
+
+import de.fu_berlin.inf.ag_se.browser.extensions.IJQueryBrowser;
+import de.fu_berlin.inf.ag_se.browser.html.ISelector;
+import de.fu_berlin.inf.ag_se.browser.html.ISelector.NameSelector;
+import java.util.ArrayList;
+import java.util.List;
+
+ public class JQueryHelper {
+  private final IJQueryBrowser browser;
+
+   public JQueryHelper(IJQueryBrowser browser) {
+    this.browser = browser;
+  }
+
+   public void clickOnSelection(ISelector selector) {
+    this.browser.run(String.format("%s[0].click()", selector.getStatement()));
+  }
+
+   public String getTextOfSelection(ISelector selector) {
+    return (String)
```

Comment:
The usage of this call in [RemoteHTMLTextElement](https://github.com/saros-project/saros/pull/480/commits/077a0fe619f8bf0c2a3ef6c4ac9b4d4fdd441481#diff-95f5543c9abb70f2995c396dbcfddaa8R18) still treats the returned value as an object of type `Object`. Either don't cast it here and adjust the return type or adjust the usage of the method.

Reply:
Adjusted the call.